docs/learn/universal-profile/metadata/edit-profile.md

```diff
@@ -293,26 +212,26 @@ async function editProfileInfo() {
 }
 ```
 
-## Step 4 - Edit the Universal Profile
+## Edit the Universal Profile
 
 Now that our updated data is encoded, we are ready to set it in our Universal Profile smart contract. To do so, we will interact with our Universal Profile smart contract via Web3.js.
 
-### 4.1 - Load an EOA
+### Load an EOA
```

Comment:
We need to remove this whole section. We are showing how to do it using the Browser Extension connected to a dApp

<img width="967" alt="image" src="https://github.com/user-attachments/assets/5faa4c41-23b4-4501-96be-1de872a4e680">